todolist/parser.go

```diff
@@ -28,6 +30,23 @@ func (p *Parser) ParseNewTodo(input string) *Todo {
 	return todo
 }
 
+func (p Parser) Parse() (string, int, string) {
+	input := p.input
+	r := regexp.MustCompile(`(\w+) (\d+) (.*)`)
```

Comment:
might be a good idea to use `\s+` instead of the fixed single whitespace between the regex groups. So if somebody types 2 spaces after `es`, it would result immediately in an error. Same for any additional whitespace after `(\d+)` which would end up in the last group and could lead to subjects with leading whitespace.